pf/tfbridge/provider_get_mapping.go

```diff
@@ -54,7 +53,7 @@ func (p *provider) GetMappingWithContext(ctx context.Context, key, provider stri
 
 func (p *provider) marshalProviderInfo(ctx context.Context) *tfbridge.MarshallableProviderInfo {
 	var providerInfoCopy tfbridge.ProviderInfo = p.info
-	providerInfoCopy.P = schemashim.ShimSchemaOnlyProvider(ctx, p.tfProvider)
+	providerInfoCopy.P = p.info.P
```

Comment:
Is this safe? p.info.P always in place?

Reply:
`p.info.P` is always in place (it's where we got `p.tfProvider` from). I'm assuming that the code looked like this since the initial bridge PF implementation passed the provider seperatily and did leave `p.info.P` as `nil`.